lib/lotus/model/adapters/memory/query.rb

```diff
@@ -554,6 +555,41 @@ def group
             raise NotImplementedError
           end
 
+          # Preload a given association into root's aggregation
+          # This should be implemented inside Repository in a class method. See example below.
+          #
+          # @since x.x.x
+          # @return Lotus::Model::Adapters::Memory::Query
+          #
+          # @example
+          #
+          #     mapping do
+          #       collections :users do
+          #         entity User
+          #         attribute :id, Integer
+          #         association :articles, [Article], foreign_key: :user_id, collection: :articles
+          #       end
+          #
+          #       collections :articles do
+          #         entity Article
+          #         attribute :id, Integer
+          #         attribute :user_id, Integer
+          #         association :user, User, foreign_key: :id, collection: :articles
+          #       end
+          #     end
+          #
+          #      class UserRepository
+          #        include Lotus::Repository
+          #
+          #       def with_articles
+          #         query.preload(:articles)
+          #       end
+          #     end
+          def preload(association)
```

Comment:
what do you think to add splat operator? I think is better this api
`preload(:users, :articles)` than `preload(:users).preload(:articles)`